test/fixtures/BootstrapCommand/npm-client-args/npm

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+
+require("fs").writeFileSync(require("path").resolve(__dirname, "npm-debug.log"), process.argv.slice(2));
```

Comment:
I don't think we need to write a file, just `console.log` the argv and assert on the command's `stdout`.